client/network/src/protocol/generic_proto/upgrade/legacy.rs

```diff
@@ -251,7 +251,7 @@ where TSubstream: AsyncRead + AsyncWrite + Unpin + Send + 'static,
 		Box::pin(async move {
 			let mut framed = {
 				let mut codec = UviBytes::default();
-				codec.set_max_len(16 * 1024 * 1024);		// 16 MiB hard limit for packets.
+				codec.set_max_len(32 * 1024 * 1024);		// 32 MiB hard limit for packets.
```

Comment:
I thought they fixed this.

Reply:
They did not modify this value. They only modified the block limits.